docs/en/enterprise-edition/policy-reference/google-cloud-policies/google-cloud-kubernetes-policies/bc-google-cloud-123.adoc

```diff
@@ -0,0 +1,81 @@
+
+== GKE container cluster configuration of NodePools is at the cluster level
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/gcp/GKEDontUseNodePools.py[CKV_GCP_123]
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is designed to identify the use of NodePools within the configuration of a Google Kubernetes Engine (GKE) cluster. The rationale behind this check is that utilizing NodePools in the cluster configuration may unnecessarily complicate the management of the cluster.
+
+When NodePools are employed at the cluster level, modifications to the node configuration will result in the creation of a new NodePool, which can increase operational complexity. Moreover, this could lead to excess resource consumption if older NodePools are not properly deleted after the creation of new ones. Therefore, it's advisable to manage node configurations outside of the cluster configuration to prevent such issues.
```

Comment:
When NodePools are used at the cluster level, modifying the node configuration creates a new NodePool, increasing operational complexity. Additionally, failing to delete older NodePools after creating new ones can lead to excess resource consumption. It is recommended to manage node configurations separately from the cluster to prevent these issues.